### PR TITLE
0x14d7ab5975c013bfadec4ef4596e95f9b672ebc9

### PR DESCRIPTION
**Pull request by BLippyBlipp**
logo

### CODE REVIEW

**Review by BLippyBlipp — COMMENTED** [review]
blippy100100

**Comment by cryptofish7** [comment]
Pls upload a logo size 120x120

**Comment by cryptofish7** [comment]
This needs to be an actual image and not a URL

**Comment by BLippyBlipp** [comment]
Heres URL : https://assets.zyrosite.com/m2Wr9P7gBjCMVNVQ/120x120-YNqN3ezwjDIQo43a.png 

120x120

**Comment by BLippyBlipp** [comment]
logo
logo

**Comment by BLippyBlipp** [comment]
logo

**Comment by BLippyBlipp** [comment]
0x14d7ab5975c013bfadec4ef4596e95f9b672ebc9
logo

**Review by BLippyBlipp — COMMENTED** [review]
logo

File: logos/0x14d7ab5975c013bfadec4ef4596e95f9b672ebc9 [hunk 1]
```diff
@@ -0,0 +1 @@
+https://assets.zyrosite.com/m2Wr9P7gBjCMVNVQ/blippy100-m2Wr9v8b9MTM6P4Z.png
```
Comment (BLippyBlipp) [inline logos/0x14d7ab5975c013bfadec4ef4596e95f9b672ebc9:1]: logo

Comment (BLippyBlipp) [inline logos/0x14d7ab5975c013bfadec4ef4596e95f9b672ebc9:1]: 120x120-YNqN3ezwjDIQo43a

**Pull request closed without merging**